config.go

```diff
@@ -0,0 +1,180 @@
+// Copyright 2016 EF CTX. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package main
+
+import (
+	"errors"
+	"fmt"
+	"net/http"
+	"regexp"
+	"strings"
+
+	"github.com/tsuru/gnuflag"
+	"github.com/tsuru/tsuru/api"
+	"github.com/tsuru/tsuru/cmd"
+	tsuruerrors "github.com/tsuru/tsuru/errors"
+)
+
+type projectConfigSet struct {
+	projectName string
+	envs        commaSeparatedFlag
+	private     bool
+	noRestart   bool
+	fs          *gnuflag.FlagSet
+}
+
+func (c *projectConfigSet) Info() *cmd.Info {
+	return &cmd.Info{
+		Name:    "config-set",
+		Desc:    "defines configuration (environment variables) for a given project",
+		Usage:   "config-set <NAME=value> [NAME=value]... [-n/--project-name projectname] [-p/--private] [--no-restart]",
+		MinArgs: 1,
+	}
+}
+
+func (c *projectConfigSet) Run(ctx *cmd.Context, client *cmd.Client) error {
+	ctx.RawOutput()
+	if c.projectName == "" {
+		return errors.New("please provide the name of the project")
+	}
+	raw := strings.Join(ctx.Args, "\n")
+	regex := regexp.MustCompile(`(\w+=[^\n]+)(\n|$)`)
+	decls := regex.FindAllStringSubmatch(raw, -1)
+	if len(decls) < 1 || len(decls) != len(ctx.Args) {
+		return errors.New("configuration vars must be specified in the form NAME=value")
+	}
+	envVars := api.Envs{
+		NoRestart: c.noRestart,
+		Private:   c.private,
+	}
+	for _, decl := range decls {
+		parts := strings.SplitN(decl[1], "=", 2)
+		envVars.Envs = append(envVars.Envs, struct{ Name, Value string }{Name: parts[0], Value: parts[1]})
+	}
+	for _, envName := range c.envs.Values() {
```

Comment:
Since project-create assume all environments when you don't pass it, I'd expect config-* to do the same. Let's wait for the opinion of the other team members before changing it, though.